build-repo.sh

```diff
@@ -132,14 +132,9 @@ fi
 # Clone packages if .rosinstall is provided
 
 ext_deps=$(find ${SRCDIR} -name "*.rosinstall" || true)
-tmp_ws=$(mktemp -d)
-touch ${tmp_ws}/.rosinstall
 for dep in ${ext_deps}; do
-  wstool merge -y -t ${tmp_ws} $dep
+  vcs import ${dep} ${ext_pkg_option} --workers 4
```

Comment:
Maybe
```suggestion
  vcs import --input ${dep} ${ext_pkg_option} --workers 4 ${extsrc}
```
(haven't tested)

Reply:
Let me give it a try. (you are definitely right about `--input`, I forgot to add it or add `<`, my bad).

Reply:
Do you think we should add `--recursive` to `ext_pkg_option`? Some repositories listed in `.rosinstall` might have submodules e.g. https://github.com/srl-freiburg/pedsim_ros.

Reply:
I think it's better to be an option which can be specified by environment variable.

Reply:
I added `VCS_OPTIONS` in 0da8f42270714e6374f544b70e3011e292c4698e.